C/manual/ch_Finance-Quote.xml

```diff
@@ -222,14 +222,21 @@ SYNOPSIS
       </step>
 
       <step>
-        <simpara>Open a Finder window, select <guimenu>Applications</guimenu> from the sidebar, double-click <guimenu>Utilities</guimenu> in the file area, then double-click on <guimenu>Terminal</guimenu> to open <application>Terminal</application></simpara>
+        <simpara>Open a Finder window, select &gm.macOS.appl; from the sidebar,
+          double-click &gmi.macOS.appl.util; in the file area, then double-click
+          on <guimenu>Terminal</guimenu> to open <application>Terminal</application>
+        </simpara>
       </step>
       <step>
-        <simpara>In the Terminal window enter <userinput>sudo /Applications/Gnucash.app/Contents/Resources/bin/gnc-fq-update</userinput>. That script asks lots of questions. Accept the default for each.
+        <simpara>In the Terminal window enter
+          <userinput>sudo /Applications/Gnucash.app/Contents/Resources/bin/gnc-fq-update</userinput>.
+          That script asks lots of questions. Accept the default for each.
         </simpara>
       </step>
       <step>
-        <simpara>Type <userinput>exit</userinput> or <keycombo action='simul'><keycap>Control</keycap><keycap>D</keycap></keycombo> to terminate the shell followed by <keycombo action='simul'><keycap>Command</keycap><keycap>Q</keycap></keycombo> to quit <application>Terminal</application>.
+        <simpara>Type <userinput>exit</userinput> or <keycombo action='simul'>&kc.ctrl;<keycap>D</keycap></keycombo>
```

Comment:
`Simul is the default value for Action` from docbook-tdg/html/keycombo.html

Perhaps we should remove `action='simul'` globally?

Reply:
see #320